packages/@ember/-internals/glimmer/lib/helpers/each-in.ts

```diff
@@ -120,7 +123,7 @@ import { consumeTag } from '@glimmer/validator';
  */
 
 /**
-  The `{{each-in}}` helper loops over properties on an object.
+  The `{{each-in}}` helper loops over properties on an object or values on sparse arrays.
```

Comment:
Conceptually, it seems pretty odd to suggest folks use `{{each-in` for arrays...